src/containers/PlaylistLiveChannels/PlaylistLiveChannels.tsx

```diff
@@ -128,19 +129,40 @@ function PlaylistLiveChannels({ playlist: { feedid, title, playlist } }: { playl
     if (channel && feedid) history.replace(liveChannelsURL(feedid, channel.id));
   }, [history, feedid, channel]);
 
-  // Loading
-  if (!channel) {
+  // Loading (channel and feedid must be defined)
+  if (!channel || !feedid) {
     return <LoadingOverlay />;
   }
 
-  const pageTitle = `${title} - ${siteName}`;
+  // SEO (for channels)
+  const canonicalUrl = `${window.location.origin}${liveChannelsURL(feedid, channel.id)}`;
+  const pageTitle = `${channel.title} - ${siteName}`;
 
   return (
     <>
       <Helmet>
         <title>{pageTitle}</title>
+        <link rel="canonical" href={canonicalUrl} />
```

Comment:
This big blob of meta tags is repeated almost identically in 3 places. Would it be better placed in the VideoDetails component?

Reply:
Yes, that's a valid point. I think we can refactor this into a single SEO/Metatags component. I will create a ticket for it.

Reply:
Created ticket #139